packages/atlas/src/components/NotificationTile/NotificationTile.tsx

```diff
@@ -0,0 +1,75 @@
+import React from 'react'
+
+import { Avatar } from '@/components/Avatar'
+import { Text } from '@/components/Text'
+import { Checkbox } from '@/components/_inputs/Checkbox'
+import { SkeletonLoader } from '@/components/_loaders/SkeletonLoader'
+
+import { AvatarWrapper, Content, Title, Wrapper } from './NotificationTile.styles'
+
+export type NotificationProps = {
+  id: string
+  author: string
+  text: string
+  date: Date | string
+  avatarUrl: string
+  videoTitle: string
+  loading?: boolean
+  onSelect?: (id: string) => void
+  read?: boolean
+  selected?: boolean
+  variant?: 'default' | 'compact'
+  className?: string
+}
+
+export const NotificationTile: React.FC<NotificationProps> = ({
+  id,
+  author,
+  text,
+  date,
+  avatarUrl,
+  videoTitle,
+  loading,
+  onSelect,
+  read,
+  selected = false,
+  variant = 'default',
+  className,
+}) => {
+  const titleVariant = variant === 'default' ? 'h300' : 't200-strong'
+  const avatarSkeletonSize = variant === 'default' ? 40 : 32
+  return (
+    <Wrapper read={read} selected={selected} loading={loading} className={className} variant={variant}>
+      {variant === 'default' && <Checkbox onChange={() => onSelect?.(id)} disabled={loading} value={selected} />}
```

Comment:
The design specifies that should be SkeletonLoader here when loading. Not disabled Checkbox. 
Yours: 
![image](https://user-images.githubusercontent.com/51168865/155291412-7551e55e-47a8-4571-8355-1baaee6e30b2.png)
Design: 
![image](https://user-images.githubusercontent.com/51168865/155291506-505b8e2e-7e16-41da-8d86-9e44c85cb5fb.png)